weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ModsPage.tsx

```diff
@@ -0,0 +1,353 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Drawer from '@mui/material/Drawer';
+import Grid from '@mui/material/Grid2';
+import TextField from '@mui/material/TextField';
+import {
+  useInsertSecret,
+  useSecrets,
+} from '@wandb/weave/common/hooks/useSecrets';
+import {TargetBlank} from '@wandb/weave/common/util/links';
+import React, {useCallback, useEffect, useMemo, useState} from 'react';
+import {useHistory} from 'react-router';
+import {Link} from 'react-router-dom';
+
+import {SimplePageLayout} from './common/SimplePageLayout';
+
+type Mod = {
+  id: string;
+  name: string;
+  description: string;
+  secrets: string[];
+};
+
+type ModCategoryType = 'Labeling' | 'Analysis' | 'Demos';
+
+type ModCategories = {
+  [key in ModCategoryType]: Mod[];
+};
+
+const mods: ModCategories = {
+  Labeling: [
+    {
+      id: 'labeling/html',
```

Comment:
seems odd to have a slash in the id

Reply:
Good eye, it is indeed odd but I did this on purpose although we may want to just not allow it.  The thinking was all of these mods will be in a git repo somewhere and organizing them in folders would be ideal.  I think I can get around this by having the id to directory mapping stored somewhere else...